src/utils.ts

```diff
@@ -114,3 +116,72 @@ export const devtools = <T extends object>(proxyObject: T, name?: string) => {
     unsub2()
   }
 }
+
+/**
+ * proxyWithComputed
+ *
+ * This is to create a proxy with initial object and additional object,
+ * which specifies getters for computed values with dependency tracking.
+ * It also accepts optional setters for computed values.
+ *
+ * [Notes]
+ * This is for expert users and not recommended for ordinary users.
+ * Contradictory to its name, this is costly and overlaps with useProxy.
+ * Do not try to optimize too early. It can worsen the performance.
+ * Mesarement and comparison will be very important.
```

Comment:
Sorry, I'm the typo guy: Mesarement -> Measurement

Reply:
That's actually helpful. Can you make it as a suggestion?